dune/loader.py

```diff
@@ -3,25 +3,38 @@
 import os
 import pickle
 import datetime
+import time
 
 
 def get_query_result(queries, query_name, cluster="medium"):
     query = queries[query_name]
 
     dune = DuneClient(os.environ["DUNE_API_KEY"])
-    pd = dune.refresh_into_dataframe(query, performance=cluster)
-    return pd
+    attempts = 0
+    while attempts < 3:
+        try:
+            pd = dune.refresh_into_dataframe(query, performance=cluster)
+            return pd
+        except:
+            attempts += 1
+            time.sleep(2)  # wait for 2 seconds before trying again
+    if attempts == 3:
+        raise Exception("Failed to get query result after 3 attempts")
+    return None
```

Comment:
exit from loop is already exception
```suggestion
    raise Exception(f"Failed to get query result after {attempts} attempts")
```